oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/web/OAuth2AuthorizationCodeGrantFilter.java

```diff
@@ -106,17 +104,17 @@
 	 * Constructs an {@code OAuth2AuthorizationCodeGrantFilter} using the provided parameters.
 	 *
 	 * @param clientRegistrationRepository the repository of client registrations
-	 * @param authorizedClientService the authorized client service
+	 * @param authorizedClientRepository the authorized client repository
 	 * @param authenticationManager the authentication manager
 	 */
 	public OAuth2AuthorizationCodeGrantFilter(ClientRegistrationRepository clientRegistrationRepository,
-												OAuth2AuthorizedClientService authorizedClientService,
```

Comment:
This is a breaking change. We should add a new constructor. The old constructor can be deprecated and just create an instance of AuthenticatedPrincipalOAuth2AuthorizedClientRepository with the provided OAuth2AuthorizedClientService

Reply:
This feature is targeted for 5.1 so we're good with the change.

Reply:
Thanks for pointing that out. That was a miss on my side that the code has never reached GA.